src/wizards/Wizard.tsx

```diff
@@ -101,6 +101,7 @@ type WizardProps = {
     previousButtonClassName?: string;
     nextButtonText?: string;
     onNext?: () => Promise<any>;
+    onNavClick?: Function;
```

Comment:
This function should be part of WizardNavLinkDetails type. As each NavLink may have different callback function

Reply:
addressed